lib/Graph/Nodes.cpp

```diff
@@ -648,6 +648,13 @@ bool ExpNode::verify() const {
   return isValid;
 }
 
+bool BucketizeNode::verify() const {
+  bool isValid = checkSameType(getInput(), getResult(), this);
```

Comment:
my understanding that output is actually index type since it's bucket number, while inputs are floats

Reply:
The output is a vector of Indices, I'll fix that!

Reply:
Fixed. I changed the output element type to `Int64ITy,      // 64-bit index type (int64_t)` and modified the node creation, lowering, verification and tests accordingly.

Reply:
Hmm a clean build and test all works on my machine. I see they are failing with
```
lib/Exporter/libExporter.a(ONNXModelWriter.cpp.o):(.data.rel.ro+0xd0): undefined reference to `glow::ONNXModelWriter::writeBucketize(glow::BucketizeNode const*, glow_onnx::GraphProto&)'
```
on the CI bots?

Reply:
ah - found the reason for that - a new PR that got merged a couple of hours ago breaking it: https://github.com/pytorch/glow/pull/3167 - will update my PR to support this exporter.

Reply:
Fixed! I added support for the newly introduced `ONNXModelWriter`

Reply:
Based on our discussion with @jfix71 I amended the commit to use 32-bit output index type:
```
Int32ITy,      // 32-bit index type (int32_t)
```

Reply:
@rdzhabarov Unrelated, as far as I can gather, OpenCL test failed? something broke on masteR?
```
The following tests FAILED:
	 20 - MLTest (Child aborted)
```

Reply:
opencl is a bit flaky. I'll restart it.